### DISCUSSION TITLE
Recent update prevents controller loading

### DISCUSSION THREAD

**Discussion by Nephenzepher** [mupen64plus-ae/mupen64plus-ae#943] [General]
After I downloaded the recent update, the load function on my controller (8 Bit Do SN 30 pro+) no longer works. I changed which button loads to verify that it is not my controller, although the controller diagnostics ensured that it was working properly. When I go in to edit my buttons, it claims that the button is still assigned, and when I press it, the load load bar on the screen lights up. I even fully charged my controller, just in case. The load still works when I pull up my menu bar.

**Comment by fzurita** [comment]
This was a recent change. Hold the load button for 3 seconds. This was to prevent accidental presses, it should probably be documented better.

**Comment by Nephenzepher** [comment]
Thank you, that worked
